kerastuner/engine/tuner.py

```diff
@@ -50,21 +51,6 @@ class Tuner(object):
         hypermodel: Instance of HyperModel class
             (or callable that takes hyperparameters
             and returns a Model instance).
-        objective: String. Name of model metric to minimize
-            or maximize, e.g. "val_accuracy".
-        max_trials: Int. Total number of trials
-            (model configurations) to test at most.
-            Note that the oracle may interrupt the search
-            before `max_trial` models have been tested.
-        executions_per_trial: Int. Number of executions
-            (training a model from scratch,
-            starting from a new initialization)
-            to run per trial (model configuration).
-            Model metrics may vary greatly depending
-            on random initialization, hence it is
-            often a good idea to run several executions
-            per trial in order to evaluate the performance
-            of a given set of hyperparameter values.
         max_model_size: Int. Maximum size of weights
```

Comment:
Note: this is Keras-specific, so the base tuner wouldn't include this argument (when we add a base tuner).

Reply:
Acknowledged